pkg/environment/applier.go

```diff
@@ -0,0 +1,178 @@
+package environment
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"os/exec"
+
+	"github.com/hashicorp/terraform-exec/tfexec"
+	"github.com/kelseyhightower/envconfig"
+)
+
+const TerraformVersion = "0.14.6"
```

Comment:
Is it not 0.14.8?
https://github.com/ministryofjustice/cloud-platform-tools-image/blob/main/Dockerfile#L6